Px.Utils/Validation/SyntaxValidation/StringValidationEntry.cs

```diff
@@ -0,0 +1,25 @@
+﻿using PxUtils.PxFile;
+
+namespace PxUtils.Validation.SyntaxValidation
+{
+    /// <summary>
+    /// Represents one line of a PX file metadata. Used for validation functions and building key-value-pair entries. This class implements the <see cref="ValidationEntry"/> interface.
+    /// </summary>
+    /// <param name="line">The line number in the file where the validation entry is located.</param>
+    /// <param name="character">The character position in the line where the validation entry starts.</param>
+    /// <param name="file">The name of the file where the validation entry is located.</param>
+    /// <param name="entry">The string that this validation entry represents.</param>
+    /// <param name="entryIndex">The index of the entry in the file.</param>
+    public class StringValidationEntry(int line, int character, string file, string entry, int entryIndex) : ValidationEntry(line, character, file)
```

Comment:
I suggest renaming StringValidationEntry to ValidationEntry and the base class ValidationEntry to ValidationItem or something. We should aim to be consistent with the terminology and only things that are actual entries should be called that.